config/environments/development.rb

```diff
@@ -6,6 +6,10 @@
   # since you don't have to restart the web server when you make code changes.
   config.cache_classes = false
 
+  config.public_file_server.enabled = true
+  # sets the asset path to either `/api/static` or `/api/approval/static`
+  config.assets.prefix = "/api#{ENV['APP_NAME'] ? "/#{ENV['APP_NAME']}/" : "/"}static"
```

Comment:
@lindgrenj6 Would suggest using `File.join`:
```ruby
config.assets.prefix = File.join(File::SEPARATOR, 'api', ENV['APP_NAME'].to_s, 'static')
```
if `ENV['APP_NAME']` is `nil` it will not be included in the path.

Reply:
Updated to use this instead, works great!